pkg/tsdb/prometheus/time_series_query_test.go

```diff
@@ -572,7 +572,7 @@ func TestPrometheus_parseTimeSeriesResponse(t *testing.T) {
 		require.NoError(t, err)
 
 		require.Len(t, res, 1)
-		require.Equal(t, res[0].Name, "legend Application")
+		require.Equal(t, res[0].Name, "")
```

Comment:
@kylebrandt i'm converting from multi-frame series to wide series in this PR. i was hoping you could take a look and give some advice on the best approach for naming the wide frame.

### main
* each frame name in the multi-frame series is set to the output of `formatLegend`
* value field has `DisplayNameFromDS` set to the output of `formatLegend`
https://github.com/grafana/grafana/blob/43c81ddd23f574ab2bba8c23e008a6cc003028f3/pkg/tsdb/prometheus/time_series_query.go#L313-L319

### this PR
* wide frame name set to `""`
* value field has `DisplayNameFromDS` set to the output of `formatLegend`

https://github.com/grafana/grafana
/blob/4984e4441292cb2bc57ede117a028d8906e0d8ef/pkg/tsdb/prometheus/framer.go#L84-L94

does that seem like the best approach?

Reply:
I honestly don't know. I thought:
 - Value Field Name is metric name
 - Labels on are also on that same field

This is what made sense to me - I never noticed the `data.TimeSeriesValueFieldName` before, that got added without me noticing. So how names get used from the Field Name vs Frame Name across Grafana is not something I know.

Reply:
I would guess it is probably okay to leave the Frame name blank on the wide frame though.

Reply:
👍 thanks. updated in 2a87f97 to use the output of `formatLegend` as the value field name

Reply:
this seemed to change the output in grafana when comparing to main, so i reverted in 7a677cd